dev/InfoBar/InfoBar.xaml

```diff
@@ -0,0 +1,203 @@
+                                        <Setter Target="ContentRoot.Visibility" Value="Collapsed"/>
+                                    </VisualState.Setters>
+                                </VisualState>
+                            </VisualStateGroup>
+                            
+                        </VisualStateManager.VisualStateGroups>
+
+                        <!-- Background is used here so that it overrides the severity status color if set. -->
+                        <Grid
+                            HorizontalAlignment="Stretch"
+                            MinHeight="{ThemeResource InfoBarMinHeight}"
+                            Background="{TemplateBinding Background}"
+                            Padding="{StaticResource InfoBarContentRootPadding}">
+
+                            <Grid.ColumnDefinitions>
+                                <ColumnDefinition Width="Auto"/> <!-- Icon -->
+                                <ColumnDefinition Width="*"/>    <!-- Title, message, and action -->
+                                <ColumnDefinition Width="Auto"/> <!-- Close button -->
+                            </Grid.ColumnDefinitions>
+
+                            <Grid.RowDefinitions>
+                                <RowDefinition Height="Auto"/>
+                                <RowDefinition Height="Auto"/>
+                            </Grid.RowDefinitions>
+
+                            <FontIcon x:Name="StandardIcon"
+                                Grid.Column="0"
+                                Visibility="Collapsed"
+                                VerticalAlignment="Top"
+                                Margin="{StaticResource InfoBarIconMargin}"
+                                FontSize="{StaticResource InfoBarIconFontSize}"
+                                Glyph="{StaticResource InfoBarInformationalIconGlyph}"
+                                FontFamily="{ThemeResource SymbolThemeFontFamily}"
+                                AutomationProperties.AccessibilityView="Raw" />
+
+                            <Border x:Name="UserIconBorder"
+                                Grid.Column="0"
+                                Visibility="Collapsed"
+                                VerticalAlignment="Top"
+                                Margin="{StaticResource InfoBarIconMargin}"
+                                Child="{Binding RelativeSource={RelativeSource TemplatedParent}, Path=TemplateSettings.IconElement}" />
+
+                            <primitives:InfoBarPanel
+                                Grid.Column="1"
+                                Margin="{StaticResource InfoBarPanelMargin}"
+                                primitives:InfoBarPanel.HorizontalMargin="{StaticResource InfoBarPanelHorizontalLayoutMargin}"
+                                primitives:InfoBarPanel.VerticalMargin="{StaticResource InfoBarPanelVerticalLayoutMargin}">
+
+                                <TextBlock x:Name="Title"
```

Comment:
Could we add `TextTrimming="CharacterEllipsis"` to the Title TextBlock here? Currently, if you add a title which cannot be line broken, we well see cut-off content which doesn't really look good at all:
![image](https://user-images.githubusercontent.com/1398851/94683665-2be57d00-0327-11eb-8fb3-73b632503a1d.png)

With text trimming:
![image](https://user-images.githubusercontent.com/1398851/94683798-65b68380-0327-11eb-8b42-e38710b8bf2d.png)

Looks much better to me.